.yamllint.yml

```diff
@@ -7,3 +7,5 @@ rules:
   line-length: disable
   truthy:
     check-keys: false
+  empty-lines:
+    max: 1
```

Comment:
That's just my two cents. Can you move the `empty-lines` config to its position in ASCII order, which would be right after the `document-start` config?
https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration

Reply:
Yeah, of course